src/apps/bridge/app_config.h

```diff
@@ -21,6 +21,12 @@ constexpr const char *SOFT_READING_PERIOD_MS = "softReadingPeriodMs";
 constexpr const char *RBR_CODA_READING_PERIOD_MS = "rbrCodaReadingPeriodMs";
 constexpr const char *TICKS_SAMPLING_ENABLED = "ticksSamplingEnabled";
 constexpr const char *TURBIDITY_READING_PERIOD_MS = "turbidityReadingPeriodMs";
+#ifdef RAW_PRESSURE_ENABLE
+constexpr const char *RBR_RAW_DIFFERENTIAL_SIGNAL_PERIOD_S = "RBRrawSampleS";
+constexpr const char *RBR_RAW_DIFFERENTIAL_BIT_DEPTH = "RBRdiffBitDepth";
+constexpr const char *RBR_MAX_RAW_REPORTS = "RBRmaxRawReports";
+constexpr const char *RBR_RAW_DEPTH_THRESHOLD_UBAR = "RBRdiffBitDepth";
+#endif // RAW_PRESSURE_ENABLE
```

Comment:
All the other strings in this file use camelCase like rbrRawSampleS, so let's make these consistent.